docs/metrics/oracle-reference.mdx

```diff
@@ -0,0 +1,51 @@
+---
+description: Reference for the available metrics in the Connect oracle
+title: Oracle
+icon: book-sparkles
+---
+
+## Dashboard Terminology
+
+- **Market:** Pair of assets that are traded against each other. For example, the BTC-USD market is the market where Bitcoin is traded against the US Dollar.
+- **Price Feed:** A price feed is indexed by a price provider and a market. For example, the Coinbase API provides a price feed for the BTC-USD market.
+- **Price Provider:** Service that provides price data for a given market. For example, the Coinbase API is a price provider for the Coinbase markets.
+- **Market Map Provider:** Service that supplies the markets that the side-car needs to fetch data for.
+
+## Exposed Metrics
+
+Connect also exposes metrics on the `/metrics` endpoint on port `8002` by default. These metrics are in the Prometheus format and can be scraped by any monitoring system that supports Prometheus format.
+
+### Health Metrics
+
+The following health metrics are emitted in Connect.
+
+- **side_car_health_check_system_updates_total:** Counter that increments every time the sidecar updates its internal state. This is a good indicator of the sidecar's overall health.
+- **side_car_health_check_ticker_updates_total:** Counter that increments every time the side-car updates the price of a given market. This is a good indicator of the overall health of a given market.
+- **side_car_health_check_provider_updates_total:** Counter that increments every time the side-car utilizes a given providers market data. This is a good indicator of the health of a given provider. Note that providers may not be responsible for every market. However, the sidecar correctly tracks the number of expected updates for each provider.
```

Comment:
I would say something about this being recommended but not required, `side_car_health_check_provider_updates_total` can be a bit noisy.

Reply:
[notify about noisyness](https://github.com/skip-mev/slinky/pull/616/commits/25ebef9f0430bfa947cbf2e240fc5a4bdfdc57b5) added a snippet about it being noisy and to consider omitting it